src/popup/locales/en.json

```diff
@@ -1244,6 +1246,16 @@
       "warningConfirm": "Are you sure you would like to proceed?",
       "cancel": "Cancel",
       "reset": "Reset"
+    },
+    "multisig-details": {
+      "address": "Multisig vault address",
+      "contract-id": "Contract id",
+      "version": "Contract version",
+      "current-nonce": "Current nonce",
+      "authorized-account": "Authorized accounts",
+      "signers": "Authorized signers:",
+      "approval": "Needed for approval:",
+      "explorer-link": "View smart contract in blockchain explorer"
```

Comment:
I suggest to use camelCase as we are moving to this naming convention within the translation files.